cluster/ci/config/roles.yaml

```diff
@@ -309,6 +309,36 @@ items:
     apiGroup: rbac.authorization.k8s.io
     name: admin
 
+- kind: Group
+  apiVersion: v1
+  metadata:
+    name: ci-search-admins
+  users:
+  - smarterclayton
+  - wking
+- kind: Role
+  apiVersion: rbac.authorization.k8s.io/v1
+  metadata:
+    namespace: ci-search
+    name: pod-admin
+  rules:
+  - apiGroups: [""]
+    resources: ["deployments", "pods", "pods/log"]
+    verbs: ["*"]
```

Comment:
`GET` and `LIST`? Why do you need to `DELETE`? :)

Reply:
> `GET` and `LIST`? Why do you need to `DELETE`? :)

I dunno, but I figured I'd need more than `GET` and `LIST` to kick things when indexing sticks, or to deploy new versions of the search tool.  We can certainly start with `GET` and `LIST` and work up from there though, if you'd rather ease into it ;).  Thoughts?

Reply:
Oh, you want to do all of that? I thought you just wanted logs. That SGTM